web_src/css/repo/reactions.css

```diff
@@ -1,91 +1,69 @@
-.segment.reactions {
+.bottom-reactions {
   display: flex;
-  border: none !important;
-  width: 100% !important;
-  max-width: 100% !important;
-  margin: 0 !important;
   gap: 6px;
-  padding: 0 1em 1em !important;
-  border-radius: 0 0 var(--border-radius) var(--border-radius);
+  margin: 0 1em 1em;
 }
 
-.timeline-item .conversation-holder .segment.reactions {
-  padding-top: 1em !important;
-  padding-bottom: 0 !important;
+.timeline-item .conversation-holder .bottom-reactions {
+  margin: 1em 0 -1em 36px; /* because there is a bottom padding on parent: "comment code-comment tw-pb-4" */
```

Comment:
Any way to avoid  the negative margin? Maybe remove the `tw-pb-4`?

Reply:
The `pb-4` was added by #15722

So, either we remove it (602c1b9fe43a0de2a0e31ef8c5eb0fafe9d5dfe1), or do other fine tunes.

Reply:
Maybe we can leave the problem to the future. Having `pb-4` or not doesn't affect the UI too much, only affects the space between the comment and "reply" buttons. But the problem is more difficult than it looks, for example: the space for "edit content", the empty `code-comment-buttons tw-flex tw-items-center tw-flex-wrap tw-mt-2 tw-mb-1 tw-mx-2` element, etc.